tests/integration/check-custom-repo/test_custom-repos.py

```diff
@@ -0,0 +1,84 @@
+import re
+import sys
+
+from collections import namedtuple
+
+import pexpect
+import pytest
+
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+from envparse import env
+
+
+def get_system_version(system_release_content=None):
+    """Return a namedtuple with major and minor elements, both of an int type.
+
+    Examples:
+    Oracle Linux Server release 6.10
+    Oracle Linux Server release 7.8
+    CentOS release 6.10 (Final)
+    CentOS Linux release 7.6.1810 (Core)
+    CentOS Linux release 8.1.1911 (Core)
+    """
+    match = re.search(r".+?(\d+)\.(\d+)\D?", system_release_content)
+    if not match:
+        return "not match"
+    version = namedtuple("Version", ["major", "minor"])(int(match.group(1)), int(match.group(2)))
+
+    return version
+
+
+@pytest.mark.good_tests
+def test_good_convertion_with_rhsm(shell, convert2rhel):
```

Comment:
Is it ok to test your feature good scenario with the rhsm?

Reply:
I'd say it can be removed. The check of the repo validity is skipped with the use of RHSM.